src/nnetbin/cuda-gpu-available.cc

```diff
@@ -41,6 +53,34 @@ int main(int argc, char *argv[]) try {
   CuDevice::Instantiate().SelectGpuId("yes");
   std::cerr
     << "### HURRAY, WE GOT A CUDA GPU FOR COMPUTATION!!! ###"
+    << std::endl << std::endl;
+  std::cerr
```

Comment:
Hi, okay, I will add there a KALDI_LOG at the very beginning. 

(Otherwise I am consistently using 'std::cerr', which is necessary in the `except() { ... }` part. I remember from lessons, that it is better not to mix the 'fprintf' with the output to std::cout std::cerr and 'fprintf' is used in  KALDI_LOG, KALDI_WARN and KALDI_ERR)